### PR TITLE
test(no-core): added missing tests for --no-core

### PR DESCRIPTION
**Pull request by sarbbottam**
@ljharb @ta2edchimp

### CODE REVIEW

File: test/bin/find.js [hunk 1]
```diff
@@ -108,4 +108,35 @@ describe('bin', function() {
...
+      '../lib/rule-finder': function(specifiedFile, noCore) {
+        return {
+          getCurrentRules: function() {
+            assert(noCore)
+            return ['current', 'rules']
+          }
+        }
+      },
+    }
+    process.argv[2] = '-c'
```
Comment (ljharb) [inline test/bin/find.js]: why use `-c` and then `--no-core`, just to demonstrate that it turns it off? perhaps a third test should be added that only does `--no-core`?

Comment (sarbbottam) [inline test/bin/find.js]: either of `c`, `a`, `u` or ` p`  is required.
https://github.com/sarbbottam/eslint-find-rules/blob/master/src/bin/find.js#L34-L39

```
if (!argv.c && !argv.p && !argv.a && !argv.u) {
  console.log('no option provided, please provide a valid option') // eslint-disable-line no-console
  console.log('usage:') // eslint-disable-line no-console
  console.log('eslint-find-rules [option] <file> [flag]') // eslint-disable-line no-console
  process.exit(0)
}
```

>test should be added that only does `--no-core`

`--no-core` w/o an option would error out.

Comment (sarbbottam) [inline test/bin/find.js]: spot on. 

What do you suggest? Should we leave it as of now and then release a breaking version with updated options?

Comment (ljharb) [inline test/bin/find.js]: That sounds like the right approach.

**Comment by sarbbottam** [comment]
Good to merge?

**Pull request merged**